baybe/surrogates/bandit.py

```diff
@@ -0,0 +1,153 @@
+"""Multi-armed bandit surrogates."""
+
+from __future__ import annotations
+
+from typing import TYPE_CHECKING, Any, ClassVar
+
+from attrs import define, field
+
+from baybe.exceptions import IncompatibleSearchSpaceError, ModelNotTrainedError
+from baybe.parameters.categorical import CategoricalParameter
+from baybe.parameters.enum import CategoricalEncoding
+from baybe.priors import BetaPrior
+from baybe.searchspace.core import SearchSpace
+from baybe.surrogates.base import Surrogate
+from baybe.targets.binary import _NEGATIVE_VALUE_COMP, _POSITIVE_VALUE_COMP
+from baybe.utils.random import temporary_seed
+
+if TYPE_CHECKING:
+    from botorch.models.model import Model
+    from botorch.posteriors import TorchPosterior
+    from torch import Tensor
+
+
+@define
+class BernoulliMultiArmedBanditSurrogate(Surrogate):
+    """A multi-armed bandit model with Bernoulli likelihood and beta prior."""
+
+    joint_posterior: ClassVar[bool] = False
+    # See base class.
+
+    supports_transfer_learning: ClassVar[bool] = False
+    # See base class.
+
+    prior: BetaPrior = field(factory=lambda: BetaPrior(1, 1))
```

Comment:
an alterantive would be to allow torch distributions here with beta as default, No need to could this to our prior classes

Reply:
Doesn't work, the logic is hardcoded to beta priors (see also [here](https://github.com/emdgroup/baybe/pull/343#discussion_r1740881331))

Reply:
i think my coment to referred to using the torch distributions here as type hint (even if its hardcoded to the beta one)
but I guess this would make problems with serialziation
in any case, more ditributions is for `the future`